CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Unreleased
 
+- Added `request_user_attention` method to `Window`.
+- **Breaking:** On macOS, removed `WindowExt::request_user_attention`, use `Window::request_user_attention`.  
+- **Breaking:** On X11, removed `WindowExt::set_urgent`, use `Window::request_user_attention`. 
```

Comment:
Could you append changes and not add them to the head, thx?

Reply:
Oh whoops, my bad. Changed it.